crypto/store/store_register.c

```diff
@@ -140,10 +140,10 @@ int ossl_store_register_loader_int(OSSL_STORE_LOADER *loader)
      *
      * scheme        = ALPHA *( ALPHA / DIGIT / "+" / "-" / "." )
      */
-    if (isalpha(*scheme))
+    if (ossl_isalpha(*scheme))
         while (*scheme != '\0'
-               && (isalpha(*scheme)
-                   || isdigit(*scheme)
+               && (ossl_isalpha(*scheme)
+                   || ossl_isdigit(*scheme)
                    || strchr("+-.", *scheme) != NULL))
```

Comment:
The side note was rather about "+-." thing. With question if one should also recognize % [that denotes hex-encoded non-ASCII character]. And with implied follow-up question *if* one recognizes %, *when* would it be handled, before or after? Which in turn can pull a chain of other follow-up questions... In other words it wasn't comment about suggested modification, but kind of "drive-by" general question. But since we are at it, as far as suggested modification goes, why not use ossl_isalnum instead?